app/send-txs.hoon

```diff
@@ -123,7 +202,10 @@
     |=  r=response:rpc:jstd
     ^-  (unit @ux)
     ?<  ?=(%batch -.r)
-    =+  txh=(tape-to-ux:ceremony (trip id.r))
+    ?<  ?=(%fail -.r)
```

Comment:
Now that we have multiple nodes, might we not want to count node connection fails as just "not confirmed"/`wait`?

Reply:
yeah, you're right.  I'll test killing one node and make sure it continues.